docs/stdlib/numbers.rst

```diff
@@ -446,11 +464,11 @@ from :eql:type:`str` and :eql:type:`json`.
 
     :index: modulo mod division
 
-    Remainder from division (modulo).
+    Retrieves the remainder from a division operation. (modulo)
```

Comment:
```suggestion
    Returns the remainder after division of the operands.
    
    This is commonly referred to as a "modulo" operation.

```
- It doesn't ring true to me that this result is "retrieved" from anywhere, so I changed to "Returns."
- Removed "operation" because it implies that the user might have to perform a division operation somehow. This happens transparently to the user, so we'll just refer to the arithmetic concept of "division" instead.
- We should mention "modulo," but I think it belongs in detail rather than in the one-liner, since the one-liner conveys what this does without it.